vizro-core/examples/visual-vocabulary/assets/css/custom.css

```diff
@@ -90,3 +90,28 @@ img[src*="#chart-icon"] {
 .intro-text ul:last-of-type {
   margin-bottom: 0;
 }
+
+.anchor-container {
+  align-items: center;
+  background: var(--text-primary);
+  border-top-left-radius: 8px;
```

Comment:
😱 A rounded border?!

Reply:
And one other thing I noticed. Do we really want the banner to block out the whole horizontal space rather than just floating on top of the underlying content?
![image](https://github.com/user-attachments/assets/03394251-0027-42c4-a943-d3cf282723ee)

Reply:
Yes 😄 I didn't like the border as is and for the banner we can make an exception 👍 

It's not the banner that blocks the whole horizontal space. The banner floats just on top of the underlying content. That horizontal space has always been there and is part of our initial layout design. It's coming from the right-side container and its padding :)